sirepo/package_data/template/common-header.py.jinja

```diff
@@ -7,3 +7,12 @@
 {% endfor %}
 """
 {% endif %}
+{% if mpi %}
+#TODO(e-carlin): discuss with rn if this is needed? It exists in sirepo.mpi but
+# it is guarded by a re of '^from __future.*' so is it only for py2?
+#TODO(e-carlin): share with sirepo.mpi
+from mpi4py import MPI
+if MPI.COMM_WORLD.Get_rank():
```

Comment:
@robnagler is this code needed in py3? The regex for a future import is throwing me off but I think it probably is needed in py3. Should it be used in slurm too? Probably can't hurt...

Reply:
a565074a0